typescript-functions-practice/get-first-element.ts

```diff
@@ -0,0 +1,5 @@
+/* exported getFirstElement */
+function getFirstElement(array: any): number {
```

Comment:
This shouldn't return a `number` because this is just returning the first element of an array. If the array was an array of `strings`, we wouldn't be allowed to return a `number` out of it! We should also specify that the type of array is an `array` of `any` and not just `any`